os/arch/arm/src/imxrt/imxrt_mpuinit.c

```diff
@@ -80,6 +81,26 @@
 #define MIN(a, b) a < b ? a : b
 #endif
 
+#if defined(CONFIG_BUILD_PROTECTED)
+const struct mpu_region_info regions_info[] = {
+	{
+		&mpu_userintsram_wb, 0x0, 0x80000000, MPU_REG_ENTIRE_MAP,
```

Comment:
Here the size of the default regions is not correct. Please refer to the memory map in os/arch/arm/src/imxrt/chip/imxrt105x_memorymap.h and set the size.

Reply:
cleaned up unnecessary changes